docker/docker-compose.override.yml.default

```diff
@@ -61,10 +61,10 @@ services:
 #    volumes:
 #      - ${DB_DATA_DIR}/${COMPOSE_PROJECT_NAME}_mysql:/var/lib/mysql:Z
 #    environment:
-#      MYSQL_DATABASE: d8
-#      MYSQL_USER: d8
-#      MYSQL_PASSWORD: d8
-#      MYSQL_ROOT_PASSWORD: d8root
+#      MYSQL_DATABASE: d9
```

Comment:
What do you think about naming these variables without including drupal version ? I don't think it serves any purpose to have `d9` here, we could have `drupal` instead

Reply:
Yep, ++ to version agnostic naming, could go separate pr

Reply:
Done here https://github.com/skilld-labs/skilld-docker-container/pull/404

Reply:
Rebased on top of it, next follow-up if for `php:81`